exporters/otlp/otlplog/otlploggrpc/go.mod

```
@@ -2,6 +2,11 @@ module go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc
 
 go 1.23.0
 
+retract (
+	// Contains broken dependency on go.opentelemetry.io/otel/sdk/log/logtest.
+	v0.12.0
```

See: https://go.dev/ref/mod#go-mod-file-retract

```suggestion
	v0.12.0 // Contains broken dependency on go.opentelemetry.io/otel/sdk/log/logtest.
```